### ISSUE TITLE
Can you help find one that works on Securely?

### ISSUE THREAD

**Issue by AstroLive8333** [xlak/hapara-delete#1]
Thanks

**Comment by AstroLive8333** [comment]
none of them do, no

**Comment by xlak** [comment]
Securly has both a Chromebook extension and a network-wide filter. Which one of them are you trying to bypass?

If it's the extension, you can use the Alphabetic exploit to hide tabs from it. (It's not supposed to completely disable the extension.) If the Chaos exploit is running but not disabling the extension, please send the extension ID here so I can add it to the list of IDs to target.

If it's the network-wide filter, you will need to find a proxy site or a VPN that isn't blocked. You can join the Titanium Network discord to find new proxy URLs. The Alphabetic exploit should also work to bypass it if the extension is active at the same time.

**Comment by xlak** [comment]
Unfortunately not at the moment. There's a fee to do that.

If your school's IT admins were actually silly enough to leave untrusted extensions enabled, there are plenty of existing Chrome extensions that let you bypass extension-based filters. One of these is [Leaf Browser](https://chrome.google.com/webstore/detail/leaf-browser/jgofbeaahhiapondaabffdhppadnimcm?hl=en).

**Comment by AstroLive8333** [comment]
Yeah, they found those. They just recently got rid of the last one I could
find, called FHS rocks.

I was gonna upload to the chrome web store, but then saw the fee :/

Do you know any other proxy sites then?

**Comment by xlak** [comment]
Try the proxy sites listed in this repo. If they don't work then join the Titanium Network discord to get new proxy sites.

**Comment by GhostieKoto** [comment]
is there a javascript bookmarklet that can delete an extension by ID or something?

**Comment by xlak** [comment]
That's what LTBEEF and Ingot do, but they're patched in version 106 of Chrome OS.

I probably should have asked way sooner, but are you using a Chromebook or another type of laptop? The Chaos exploit only works on Chrome OS.

**Comment by GhostieKoto** [comment]
Chromebook (version 107

**Comment by xlak** [comment]
I need some more information on what step you get stuck on with each bypass method. Please include screenshots if applicable.

**Comment by GhostieKoto** [comment]
![Screenshot 2022-12-09 8 55 49 AM](https://user-images.githubusercontent.com/112585351/206718198-17063b08-cff2-4f69-9a04-5419fc9fee05.png)
![Screenshot 2022-12-09 8 52 55 AM](https://user-images.githubusercontent.com/112585351/206718213-b3e77af0-4ed1-4992-b2fb-1b2ef4584679.png)
![Screenshot 2022-12-09 8 52 50 AM](https://user-images.githubusercontent.com/112585351/206718214-b49505af-6431-487a-8114-2bc2b6194d7d.png)
![Screenshot 2022-12-09 8 51 22 AM](https://user-images.githubusercontent.com/112585351/206718217-25791d09-38be-4683-8cde-4ba151f02f69.png)
![Screenshot 2022-12-09 8 50 29 AM](https://user-images.githubusercontent.com/112585351/206718221-de54cb5e-689a-4da9-aa2b-6bbb1eab7319.png)

There you go

Our administrator recently banned all extensions and made it so you have to request the extensions instead of downloading them. What do I do? Do you think it would affect him much if I spammed him with requests? Or would that just make him stop doing requests and stop extensions altogether? Then again, if he makes it so you can't get extensions, maybe then the students and teachers would say something about it, but I doubt it. any ideas on how to make his life miserable? (Not by hurting him, but by using tech to annoy the fvck out of him, until he gives us our internet rights back)

**Comment by xlak** [comment]
Wow, everything's extremely locked down. IMO sysadmins should not be responsible for being the "no fun police" and spending so much effort on stopping you from playing online games when you're finished all your work. Web filters are meant to protect students from accidentally coming across harmful content, and it's your decision to bypass them. It's a classroom management issue to be enforced by teachers if the school is not OK with that.

Here are some interesting ideas on how to troll your administrator:

- Rotate the screens on chromebooks upside down (Press control+shift+refresh twice, do it again to revert it)
- Spam requests for extensions that sound like they might exploits but are actually harmless (Unicorn Apocalypse, NoGuardian, React Dev Tools, etc)
- Spam requests for extensions that are already unblocked
- Get lots of people to go on blocked proxy websites, it will fill up the logs and make him wonder if anyone found an unblocked one
- Turn on ChromeVox from the login screen on the chromebooks
- Disable the touchpads on the chromebooks (Search+Shift+P)

I'm currently analyzing the Securly source code to make the Alphabetic exploit be able to bypass it, but any additional details you can give will help tremendously. Does the Alphabetic exploit fail right after you click the bookmark or at a later step?

**Comment by GhostieKoto** [comment]
The nebula proxy doesn't work too well, mostly because it mostly returns 504 and 401 (insufficient perms (401) or cannot connect (504))

How will rotating the screen affect him? (they don't watch our screens)
I'm thinking of spamming requests
If I go on blocked proxy websites, will that do the same or does it have to be from multiple ip addresses?
What will ChromeVox do against him?
and what will disabling touchpads do against him?

Do you think you could send me the Securely source code?

I know that Securely can't touch files, so that's one way to play games. I know a few game website if you want them. I'm actively looking, so if you wanna recruit me I'd be happy to send you anything I find.

As of right now, I have quite a few gaming websites, and like, 2 proxy websites

Hapara.cf
tan-misty-reindeer.cyclic.app

Those are the only 2 I have that work

**Comment by xlak** [comment]
Your sysadmin having to fix messed up chromebooks with flipped screens or the touchpads not working will leave less time to focus on blocking things. Turning on ChromeVox on the lock screen [will make it even more annoying to deal with](https://www.reddit.com/r/k12sysadmin/comments/vd005w/do_you_want_to_activate_chrome_vox/).

What websites are broken on the Nebula proxy? I can work on fixing them.

I can make a private repo with the Securly source code (so it doesn't get taken down due to copyright.)

Feel free to open pull requests on this repo to add any info you find.

**Comment by xlak** [comment]
[Here is an existing repo with the source code for Securly](https://github.com/TN-Decompile/Securly-Decompile)

**Comment by GhostieKoto** [comment]
I have good news and bad news...

Good news, I don't have to request extensions anymore

Bad news, they're now blocking a lot more websites, including ones that I use for coding, such as cyclic.app, I'm assuming because we have too much freedom (programming proxies)
